org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -2313,7 +2349,10 @@ class DocumentOnTypeFormattingParams extends DocumentFormattingParams {
  * The document range formatting request is sent from the client to the server to format a given range in a document.
  */
 @JsonRpcData
-class DocumentRangeFormattingParams extends DocumentFormattingParams {
+class DocumentRangeFormattingParams extends DocumentFormattingParams implements WorkDoneProgressParams {
```

Comment:
Redundant super type? The spec say `interface DocumentFormattingParams extends WorkDoneProgressParams {..}` so DocumentRangeFormattingParams should not need the super type again.

Reply:
It's the previous code of the protocol. Yes your are right.
We can change it here, or a create new merge request dedicated., up to you